kata-deploy/README.md

```diff
@@ -116,7 +116,7 @@ In order to use a workload Kata with Firecracker, first add a `RuntimeClass` as:
   $ kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/master/kata-deploy/k8s-1.13/kata-fc-runtimeClass.yaml
   ```
 
-The following YAML snippet shows how to specify a workload should use Kata with QEMU:
+The following YAML snippet shows how to specify a workload should you use Kata with QEMU:
```

Comment:
urgh, I know you inherited these sentences @nitkon :-), but they make me twitch (not your fault).
Let's see if we can improve them... if I take the first similar instance I found in the doc:

```
In order to use a workload Kata with QEMU
```

should probably be something simpler like:

```
To use a workload with Kata-QEMU
```

Similarly, I suspect the line you edited could be:

```
The following YAML snippet shows how to specify a workload using Kata-QEMU
```

Are you up for hunting down other instances in the doc?
/cc @jodh-intel for extra grammaticalness.....

Reply:
Agreed, that would be great if you don't mind @nitkon ? (good to see you again by the way! :smile:)

Reply:
@jodh-intel @grahamwhaley : Fixed! ;-)